tests/model_serving/model_server/kueue/conftest.py

```
@@ -12,18 +13,77 @@
+    try:
+        csvs = list(
+            ClusterServiceVersion.get(
+                dyn_client=admin_client,
+                namespace=py_config.get("applications_namespace", "openshift-operators"),
+            )
+        )
+        for csv in csvs:
+            if csv.name.startswith("kueue") and csv.status == csv.Status.SUCCEEDED:
+                BASIC_LOGGER.info(f"Found Kueue operator CSV: {csv.name}")
+                return True
+        return False
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)